src/collections/utils.ts

```diff
@@ -121,42 +119,21 @@ export async function createStargateUri (
     if (logLevel) {
         uri.searchParams.append('logLevel', logLevel);
     }
-    const accessToken = await getStargateAccessToken(baseAuthUrl, username, password);
+    const accessToken = await getStargateAccessToken(username, password);
     uri.searchParams.append('applicationToken', accessToken);
     return uri.toString();
 }
 
 /**
  * Get an access token from Stargate (this is useful while connecting to open source JSON API)
- * @param authUrl the base URL of the JSON API auth (this is generally the Stargate Coordinator auth URL)
  * @param username Username
  * @param password Password
  * @returns access token as string
  */
 export async function getStargateAccessToken(
-    authUrl: string,
     username: string,
     password: string) {
-    try {
-        const response = await axios({
-            url: authUrl,
-            data: { username, password },
-            method: 'POST',
-            headers: {
-                Accepts: 'application/json',
-                'Content-Type': 'application/json'
-            }
-        });
-        if (response.status === 401) {
-            throw new StargateAuthError(response.data?.description || 'Invalid credentials provided');
-        }
-        return response.data?.authToken;
-    } catch (e: any) {
-        if (e.response?.data?.description) {
-            e.message = e.response?.data?.description;
-        }
-        throw e;
-    }
+    return 'Cassandra:' + Buffer.from(username).toString('base64') + ':' + Buffer.from(password).toString('base64');
```

Comment:
One minor detail: can we make `getStargateAccessToken()` not async anymore? It doesn't do any async operations, so marking `getStargateAccessToken()` as async just creates an unnecessary promise. If you make this change, also ideally remove `await` from any place where we do `await getStargateAccessToken()`